packages/screens/Rakki/RakkiScreen.tsx

```diff
@@ -0,0 +1,857 @@
+  if (rakkiInfo === undefined) {
+    content = (
+      <View style={{ height }}>
+        <LoaderFullSize />
+      </View>
+    );
+  } else if (rakkiInfo === null) {
+    content = (
+      <View
+        style={{
+          flex: 1,
+          width: "100%",
+          alignItems: "center",
+          justifyContent: "center",
+        }}
+      >
+        <BrandText style={sectionLabelCStyle}>
+          RAKKi is not deployed on this network
+        </BrandText>
+      </View>
+    );
+  } else {
+    content = (
+      <>
+        <RakkiLogo style={{ marginTop: 100 }} />
+        <PrizeInfo
+          info={rakkiInfo}
+          networkId={networkId}
+          style={{ marginTop: 50 }}
+        />
+        <GetTicketCTA info={rakkiInfo} style={{ marginTop: 32 }} />
+        <GameBox
+          info={rakkiInfo}
+          networkId={networkId}
+          style={{ marginTop: 32 }}
+        />
+        <Help style={{ marginTop: 50 }} />
+        <History
+          info={rakkiInfo}
+          networkId={networkId}
+          style={{ marginTop: 50 }}
+        />
+      </>
+    );
+  }
+  return (
+    <ScreenContainer
+      footerChildren={rakkiInfo === undefined ? <></> : undefined}
+    >
+      {content}
+    </ScreenContainer>
+  );
+};
+
+const BuyTicketsButton: React.FC<{ networkId: string; info: Info }> = ({
+  networkId,
+  info,
+}) => {
+  const selectedWallet = useSelectedWallet();
+  const [modalVisible, setModalVisible] = useState(false);
+  const remainingTickets = info.config.max_tickets - info.current_tickets_count;
+  const [ticketAmount, setTicketAmount] = useState("1");
+  const queryClient = useQueryClient();
+  const ticketAmountNumber = Long.fromString(ticketAmount || "0");
+  useEffect(() => {
+    if (remainingTickets > 0 && ticketAmountNumber.gt(remainingTickets)) {
+      setTicketAmount(remainingTickets.toString());
+    }
+  }, [ticketAmountNumber, remainingTickets]);
+  const totalPrice = ticketAmountNumber.mul(
+    Long.fromString(info.config.ticket_price.amount),
+  );
+  const { balances } = useBalances(networkId, selectedWallet?.address);
+  const ticketDenomBalance =
+    balances.find((b) => b.denom === info.config.ticket_price.denom)?.amount ||
+    "0";
+  const canPay = Long.fromString(ticketDenomBalance).gte(totalPrice);
+  const canBuy = ticketAmountNumber.gt(0) && canPay;
+  const { wrapWithFeedback } = useFeedbacks();
+  return (
+    <View style={{ flexDirection: "row" }}>
+      <SecondaryButton
+        onPress={() => setModalVisible(true)}
+        textStyle={{ fontWeight: "400" }}
+        text="Buy tickets"
+        size="XS"
+      />
+      <ModalBase
+        scrollable
+        label="Buy RAKKi tickets!"
+        visible={modalVisible}
+        onClose={() => setModalVisible(false)}
+      >
+        <Box
+          notched
+          style={{
+            backgroundColor: "#171717",
+            borderWidth: 1,
+            borderColor: "#333",
+            padding: 12,
```

Comment:
if we had serious color naming and design consistence + a theme system it would be more important

but I don't understand the advantage of using `neutral17` vs `"#171717"`

Reply:
Same